cmd/storage-rest-server.go

```diff
@@ -148,6 +149,7 @@ func (s *storageRESTServer) AppendFileHandler(w http.ResponseWriter, r *http.Req
 		s.writeErrorResponse(w, err)
 		return
 	}
+	xnet.GlobalNetStats.Receive(uint64(r.ContentLength))
```

Comment:
we do have `globalConnStats` that holds input and output bytes per server, is it possible to use that instead of adding another stat tracking layer..?

Reply:
We’re tracking successful communication between nodes, with distinction between data and control transmissions. Doing this at the conn level makes it impossible to do that

Reply:
I think perhaps what @nitisht meant is globalConnStats is not needed anymore with this implementation, we should remove it @wlan0 ?  -

I can remove it in the HTTP 2.0 PR though. But we should expose these values over Prometheus for existing users.

Reply:
> I think perhaps what @nitisht meant is globalConnStats is not needed anymore with this implementation, we should remove it @wlan0 ? -

Yeah we should have single approach handle net stats, even if not done in this PR

Reply:
We can do that perhaps in the next iteration there is another change needed i.e HTTP 2.0 support cleanup the globalConnStats information @nitisht  - this PR can become too big with lots of unrelated changes.